jib-core/src/main/java/com/google/cloud/tools/jib/event/events/LogEvent.java

```diff
@@ -118,4 +118,9 @@ public boolean equals(Object other) {
   public int hashCode() {
     return Objects.hash(level, message);
   }
+
+  @Override
+  public String toString() {
```

Comment:
Leftover from debugging? (Actually maybe it'd be useful to add `toString`s to the rest of the events in another PR?)

Reply:
I had mentioned adding it as it was useful for debugging tests.  But I forgot that we have other instances, so I've backed it out.

Reply:
Ah, I should read the PR description next time :man_facepalming: